courses/offering/DataFormat.jsx

```diff
@@ -0,0 +1,75 @@
+/**
+ * Converts boolean values into emojis.
+ * @param {boolean} p - a boolean value that is either true or false.
+ * @returns {string} a string of emoji icon signifying true or false. If 'p' is neither true or false, returns an error message.
+ */
+function BooleanToEmoji(b) {
+    if (b === true) return "✅";
+    else if (!b) return "❌";
+    return "ERROR";
+}
+
+/**
+ * Converts a string of UWindsor undergrad CS program into emojis.
+ * @param {string} p - an acronym of one of the UWindsor CS programs.
+ * @returns {string} a string of emoji icon signifying the program.
+ */
+function ProgramToEmoji(p) {
+    if (p === "CSG") return "📚";
+    else if (p === "CSH") return "🎓";
+    else if (p === "CSHAC") return "📲";
+    else if (p === "CSSE") return "⚙";
+    else if (p === "BACS") return "👔";
+    else if (p === "MCS") return "📐";
+    return p;
+}
+
+/**
+ * Converts a string array of UWindsor CS program acronyms and into an array of emojis
+ * @param {string} programs - an array of acronyms of one of the UWindsor CS programs.
+ * @returns {string} text - a string array of emoji icons signifying the programs.
+ */
+function ListProgramsAsEmoji(programs) {
+    var text = "";
+    const len = programs.length;
+    for (let i = 0; i < len; i++) {
+        text += ProgramToEmoji(programs[i]);
+    }
+    return text;
+}
+
+/**
+ * Joins an array of string elements with commas.
+ * @param {string} text - a string array of words.
+ * @returns {string} out - a string of words separated by commas.
+ */
+function JoinWithComma(text) {
```

Comment:
Why was this function created? Assuming `text` is an array, you can use
```js
text.join(', ');
```
instead.